automationexcersise/config.json

```diff
@@ -3,7 +3,9 @@
     "base_url": "http://automationexercise.com",
     "wait_duration": 5,
     "email": "johnsmith123321@gmail.com",
-    "password": "johnsmith321123"
+    "password": "johnsmith321123",
+    "inc_email": "abcdef@abc.ab",
+    "inc_password": "abcdef123"
```

Comment:
let's not keep credentials in the repo since you've learned a better way of keeping secrets.

Reply:
Changed to environment variables